css-layout-classes/styles.css

```diff
@@ -0,0 +1,32 @@
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+  flex-direction: column;
+  font-family: sans-serif;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+}
+
+.column-one-third {
+  display: flex;
+  max-width: calc(100% / 3);
+  flex-direction: column;
+}
+
+.column-two-thirds {
+  display: flex;
+  flex-direction: column;
+  max-width: calc(2 * (100% / 3));
+}
+
+.column-one-fourth {
+  display: flex;
+  max-width: calc(100% / 4);
```

Comment:
Rather than using `max-width`, we want to use `width`. `max-width` is still letting your content dictate the size of the column, but you cut it off after a certain point, but we want to force our columns to be a certain size